app/styles/_content.scss

```diff
@@ -0,0 +1,75 @@
+.marketing{
+  height: 150px;
+}
+
+.mark-shoe{
+  top: 50%;
+  -ms-transform: translateY(-50%);
+  -webkit-transform: translateY(-50%);
+  transform: translateY(-50%);
+}
+
+.img-shoe{
+  height: 6em;
+  width: 12em;
+}
+
+.mark-tshirt{
+  top: 50%;
+  -ms-transform: translateY(-50%);
+  -webkit-transform: translateY(-50%);
+  transform: translateY(-50%);
+}
+
+.img-tshirt{
+  height: 9em;
+  width: 9em;
+}
+
+.mark-short{
+  top: 50%;
+  -ms-transform: translateY(-50%);
+  -webkit-transform: translateY(-50%);
+  transform: translateY(-50%);
+}
+
+.img-short{
+  height: 10em;
+  width: 9em;
+}
+
+.title{
+  height: 70px;
+  background-color: $turquoise1;
+  color: $white;
+}
+
+.products{
+  height: 35em;
+  margin-top: 3em;
+}
+
+.product{
+  height: 200px;
+  width: 200px;
+  border: 2px solid $grey;
+  border-bottom: 5px solid $turquoise1;
+  text-align: center;
+}
```

Comment:
We can't use a fixed width when working with a responsive design. Please remove that width, and instead use the col-md-# that you think it's appropriate.